backend/src/test/java/com/ody/member/service/MemberServiceTest.java

```diff
@@ -93,4 +109,23 @@ private DeviceToken getDeviceTokenByAuthProvider(String providerId) {
             return memberRepository.findByAuthProvider(new AuthProvider(providerId)).get().getDeviceToken();
         }
     }
+
+    @DisplayName("회원을 삭제한다.")
+    @Test
+    void delete() {
+        Member member = fixtureGenerator.generateMember();
+        Meeting meeting = fixtureGenerator.generateMeeting();
+        Mate mate = fixtureGenerator.generateMate(meeting, member);
+        Eta eta = fixtureGenerator.generateEta(mate);
+        Notification notification = fixtureGenerator.generateNotification(mate);
+
+        memberService.delete(member.getId());
+
+        assertAll(
+                () -> assertThat(memberRepository.findById(member.getId())).isEmpty(),
+                () -> assertThat(mateRepository.findAllByMemberId(member.getId())).isEmpty(),
+                () -> assertThat(etaRepository.findById(eta.getId())),
```

Comment:
```suggestion
                () -> assertThat(etaRepository.findById(eta.getId())),
```

이 라인은 혹시 어떤 걸 테스트 하는 것인가요?

Reply:
[반영] 요즘 눈이 침침하네요 [3d36048](https://github.com/woowacourse-teams/2024-ody/pull/512/commits/3d36048bd366e95401862b927f622de431386397)